src/components/Navbar/Navbar.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { Navbar } from './Navbar';
+
+const content = 'navbar content';
+
+storiesOf('Navbar', module)
+	.addParameters({ jest: ['Container'] })
+	.add('Navbar', () => (
+		<Fragment>
+			<Navbar>{content}</Navbar>
+			<div style={{ height: '200px' }} />
+			<Navbar placement="bottom">{content}</Navbar>
+		</Fragment>
+	))
+	.add('Fixed navbar', () => (
+		<Fragment>
+			<Navbar position="fixed" background={'alt'}>
+				{content}
+			</Navbar>
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
+			<br />
+			lorem
```

Comment:
Beetje slordig imo. Moet dit hier staan om aan te tonen dat die navbar fixed is? Als je dan toch een pagina wil vullen, gebruik een spacer of de `lorem-ipsum`-package die al in het project zit.